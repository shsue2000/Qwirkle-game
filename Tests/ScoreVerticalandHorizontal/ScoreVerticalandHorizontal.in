2
ScoreVerticalandHorizontal.save
place O1 at B1
place P1 at C1
place O2 at B2