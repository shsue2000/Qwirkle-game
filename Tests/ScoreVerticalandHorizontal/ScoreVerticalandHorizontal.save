APPLE
0
R3,O1,O2,Y1,P2,O5
PEAR
0
B3,B6,P1,R6,G5,R5
3,3

R5,P2,R4,R2,Y2,B2,Y3,P6,R5,B2,O3,P5,B5,G4,G2,G1,Y4,P6,Y6,G4,Y2,P4,B1,R6,Y3,G2,P3,Y4,R1,O2,R4,B4,O5,O6,G1,B5,R1,G3,B4,Y5,O4,G3,R3,Y6,G6,P1,R2,O3,O6,B3,O4,G5,G6,Y5,B1,P5,O1,P3,P4,B6
APPLE