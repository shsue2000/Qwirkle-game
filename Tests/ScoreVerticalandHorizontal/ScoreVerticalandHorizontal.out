Welcome to Quirkle!
-------------------
Menu
----
1. New Game
2. Load Game
3. Credits (Show student information)
4. Quit
> 2        
Enter the filename from which load a game
curr
Loading: curr
Qwirkle game successfully loaded
APPLE, it's your turn
Score for APPLE: 0
Score for PEAR: 0
   0  1  2
  ----------
A |  |  |  |
B |  |  |  |
C |  |  |  |
Your hand is
R3,O1,O2,Y1,P2,O5
> place O1 at B1
PEAR, it's your turn
Score for APPLE: 0
Score for PEAR: 0
   0  1  2
  ----------
A |  |  |  |
B |  |O1|  |
C |  |  |  |
Your hand is
B3,B6,P1,R6,G5,R5
> place P1 at C1
APPLE, it's your turn
Score for APPLE: 0
Score for PEAR: 3
   0  1  2
  ----------
A |  |  |  |
B |  |O1|  |
C |  |P1|  |
D |  |  |  |
Your hand is
R3,O2,Y1,P2,O5,P2
> place O2 at B2
PEAR, it's your turn
Score for APPLE: 3
Score for PEAR: 3
   0  1  2  3
  -------------
A |  |  |  |  |
B |  |O1|O2|  |
C |  |P1|  |  |
D |  |  |  |  |
Your hand is
B3,B6,R6,G5,R5,R4